pipelines/containerfiles/Containerfile.openvino.mlserver.mlflow

```diff
@@ -18,15 +18,8 @@ RUN mkdir /models && chown ovms:ovms /models
 # CHANGE THIS LINE TO MATCH YOUR MODEL
 COPY --chown=ovms:ovms $MODEL_DIR /models/1
 
-RUN <<EOF
-    # Not deleting this file causes a bug
-    if echo "${MODEL_DIR}" | grep -q "tf2model"; then rm -f /models/1/fingerprint.pb; fi
-
-    chmod o+rwX /models/1
-
-    # https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
-    chgrp -R 0 /models/1 && chmod -R g=u /models/1
-EOF
+                                # Not deleting this file causes a bug                                                # https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
+RUN if echo "${MODEL_DIR}" | grep -q "tf2model"; then rm -f /models/1/fingerprint.pb; fi && chmod o+rwX /models/1 && chgrp -R 0 /models/1 && chmod -R g=u /models/1
```

Comment:
Does anyone remember why we have to remove `fingerprint.pb` from the file?

Is setting `chmod o+rwX` required?  This would open permissions to anyone user with access?

Not a reason to block this PR as everything can be addressed in a follow-up PR

Reply:
@LaVLaS I don't know.

Reply:
Good questions @LaVLaS...we should see if we can simplify things here. I'm not sure what the answers currently are.